### PR TITLE
Facebook login in signup page

### PR DESCRIPTION
**Pull request by MechanicKim**
I fixed sing-in problem in SignUp page

### CODE REVIEW

**Comment by ace0625** [comment]
👍

File: src/components/Login/SignUp.js [hunk 1]
```diff
@@ -109,9 +110,32 @@ class SignUp extends Component {
 
   onLoginCallback(result, error) {
     if (error) {
-      this.alert('Sever error(Profile)! Please sign in again.');
+      this.alert('Please check your account information and sign in again!');
```
Comment (SarahhhC) [inline src/components/Login/SignUp.js:113]: When this err happen?

Comment (MechanicKim) [reply src/components/Login/SignUp.js:113]: This logic is same with a login in Login.js
I just moved this logic into SignUp.js

This error is about Facebook sign-in.
But, this error is occurred after calling signing api.
You can refer this on line 30 in UserUtil.js.

Comment (MechanicKim) [reply src/components/Login/SignUp.js:113]: I think this is not reasonable.
We'll have to resolve it.

Comment (SarahhhC) [reply src/components/Login/SignUp.js:113]: Ok, maybe not this time :)

**Comment by SarahhhC** [comment]
LGTM, thanks for fixing 👍 

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/102/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**